ros_buildfarm/status_page.py

```diff
@@ -39,6 +39,111 @@
 from .templates import get_template_path
 
 
+def build_blocked_source_entries_page(
```

Comment:
Maybe move below the existing function `build_blocked_releases_page`?

Reply:
Moved in 5d615c3